packages/create-bison-app/template/package.json.ejs

```diff
@@ -21,21 +18,20 @@
     "db:seed": "yarn prisma db seed",
     "db:seed:prod": "cross-env APP_ENV=production prisma db seed",
     "db:setup": "yarn db:reset",
-    "dev": "concurrently -n \"WATCHERS,NEXT\" -c \"black.bgYellow.dim,black.bgCyan.dim\" \"yarn watch:all\" \"next dev\"",
+    "dev": "next dev",
```

Comment:
We don't need to run TypeScript concurrently with our dev server; IDEs take care of TypeScript for us.